packages/nextra/src/hooks/use-page-context.ts

```diff
@@ -0,0 +1,41 @@
+import { useRouter } from 'next/router'
```

Comment:
Test fails because we compile hooks to mjs extension, we should compile hooks to js or add .js extension here